components/test_util/src/logging.rs

```diff
@@ -98,24 +98,10 @@ pub fn init_log_for_test() {
     .unwrap();
     let writer = output.map(|f| Mutex::new(File::create(f).unwrap()));
     // we don't mind set it multiple times.
-    let drain = CaseTraceLogger { f: writer };
+    let drainer = CaseTraceLogger { f: writer };
 
-    // Collects following targets.
-    const ENABLED_TARGETS: &[&str] = &[
-        "tikv::",
-        "tests::",
-        "benches::",
-        "integrations::",
-        "failpoints::",
-        "raft::",
-        // Collects logs for test components.
-        "test_",
-    ];
-    let filtered = drain.filter(|record| {
-        ENABLED_TARGETS
-            .iter()
-            .any(|target| record.module().starts_with(target))
-    });
+    // Default disabled log targets for test.
+    let disabled_targets = vec!["tokio_core".to_owned(), "tokio_reactor".to_owned()];
```

Comment:
we may need to disable these targets by default, no matter in test or in production.

Reply:
there are only debug logs for these mods, seems it's okay for production?

Reply:
I think it's still better to disable them by default, because sometimes we will need debug level logs in production to trace some problems and then these modules will mess up our logs.

Reply:
BTW a kindly reminder: if we add these default disabled targets, then the empty check shall be removed.

Reply:
If that you can set env variable, And I want to avoid string comparing in production for performance issue.

Reply:
Ok this is a good reason for not enabling them by default. However our DBAs may not be happy with this XD. I will give my approval after testing this PR personally.

Reply:
thx!